src/pages/dashboard/Explore.tsx

```diff
@@ -56,7 +83,15 @@ const Explore: FC = () => {
           </span>
         )}
       </div>
-
+      {searchResult?.numFound && searchResult.numFound > NumberOfHitsPrPage && <Typography>Page: {page}</Typography>}
+      {searchResult?.numFound && searchResult.numFound > NumberOfHitsPrPage && (
```

Comment:
Lurer på om backenden støtter offset og paginering slik at vi slipper å filtrerere resultatet selv.

https://api-dev.dlr.aws.unit.no/dlr-gui-backend-resources-search/v1/resources/search?filter=facet_type%3A%3Alearning&order_by=created&order=desc&limit=20&offset=0&mine=false&showInaccessible=false&query=fotosession

Reply:
Skjønte ikke helt den. Det er ingen filtrering av resultatet på frontend. Logikken på linje 87 er kun for å vise/skjule pagineringskomponenten etter hvor mange treff man har.